.github/workflows/build-publish.yaml

```diff
@@ -0,0 +1,33 @@
+#
+# Copyright (c) 2021 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+on:
+  push:
+    paths-ignore:
+    - '**.md'
+jobs:
+  build:
+    env:
+      IMAGE_NAME: quay.io/eclipse/che-backup-server-rest
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone source code
+        uses: actions/checkout@v1
+        with:
+          fetch-depth: 1
+      - name: Docker login
+        uses: azure/docker-login@v1
+        with:
+          login-server: quay.io
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Build and push image
+        run: |
+          TAG=$(git rev-parse --short HEAD)
+          docker build -t ${IMAGE_NAME}:${TAG} .
```

Comment:
Are we going to provide multi-platform build?

Reply:
Ok, I will